parsers/verilog.c

```diff
@@ -386,18 +453,23 @@ static void buildKeywordHash (const langType language, unsigned int idx)
 		if (p->isValid [idx])
 			addKeyword (p->keyword, language, (int) p->kind);
 	}
+	/* mark unspecified keywords as K_IGNORE */
+	for (i = 0  ;  i < size  ;  ++i) {
+		if (lookupKeyword(keywords[i], language) == KEYWORD_NONE)
```

Comment:
I wonder why this `lookupKeyword` is needed at runtime?
If you want to detect adding the same keyword twice, Assert can be used.
Assert is enabled when you run ./configure with --enable-debugging.
On Travis-CI, ctags is built with the option.

Reply:
I withdraw the above comment.  I understand the intention.